actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -178,8 +178,8 @@ def use_cookies_with_metadata
   #   only HTTP. Defaults to +false+.
   # * <tt>:same_site</tt> - The value of the +SameSite+ cookie attribute, which
   #   determines how this cookie should be restricted in cross-site contexts.
-  #   Possible values are +nil+, +:none+, +:lax+, and +:strict+. Defaults to
-  #   +:lax+.
+  #   Possible values are +:none+, +:lax+, +:strict+, and +false+. Defaults to
+  #   +:lax+. If +false+ is provided, no +SameSite+ attribute will be set.
```

Comment:
Since `nil` will work now, I would be in favor of leaving this as is.  `nil` does a good job of implying "not present" with respect to "the value of the `SameSite` cookie attribute".